tests/python/unittest/test_operator.py

```diff
@@ -17,6 +17,7 @@
 
 # pylint: skip-file
 from __future__ import print_function
+from __future__ import division
```

Comment:
It seems that it is unused? The types of variables are specific as float or int.